joomla/joomla-cms
# [4.3] Smart Search: Hiding language taxonomy without multilanguage enabled

Pull Request for Issue #39390.

### Summary of Changes
When the site is not multilanguage, the language taxonomy in Smart Search should not be displayed. Also, don't display the language '*'.

### Testing Instructions
Look at search results in the frontend and see the taxonomy "Language" on a monolingual site. Apply the patch and see the language disappear. Enable the languagefilter plugin and see the language taxonomy only apear for items which have a language defined.

### Link to documentations
Please select:
